tools/cli/datalab.py

```diff
@@ -170,6 +192,12 @@ def run():
                 dest='zone',
                 default=None,
                 help=_ZONE_HELP)
+        if command_config['require-email']:
+            subcommand_parser.add_argument(
+                '--email',
+                dest='email',
+                default=get_email_address(),
```

Comment:
This wouldn't default to a service account unless the user runs `datalab create` from inside of a VM, and that seems like a rare scenario.